components/stack_exchange/sources/new-answers-for-questions/new-answers-for-questions.js

```diff
@@ -0,0 +1,74 @@
+const stack_exchange = require('../../stack_exchange.app');
+
+module.exports = {
+  key: "stack_exchange-new-answers-for-questions",
+  name: "New Answers for Specific Questions",
+  description: "Emits an event when a new answer is posted in one of the specified questions",
+  version: "0.0.1",
+  dedupe: "unique",
+  props: {
+    stack_exchange,
+    db: "$.service.db",
+    siteId: { propDefinition: [stack_exchange, "siteId"] },
+    questionIds: {
+      propDefinition: [
+        stack_exchange,
+        "questionIds",
+        c => ({ siteId: c.siteId }),
+      ],
+    },
+    timer: {
+      type: '$.interface.timer',
+      default: {
+        intervalSeconds: 60 * 15, // 15 minutes
+      },
+    },
+  },
+  hooks: {
+    async activate() {
+      const fromDate = this._getCurrentEpoch();
+      this.db.set("fromDate", fromDate);
+    },
+  },
+  methods: {
+    _getCurrentEpoch() {
+      // The StackExchange API works with Unix epochs in seconds.
+      const epochInMs = +new Date();
+      return Math.floor(epochInMs / 1000);
+    },
+    generateMeta(data) {
+      const {
+        answer_id: id,
+        question_id: questionId,
+        creation_date: ts,
+      } = data;
+      const summary = `New answer for question ID ${questionId}`;
+      return {
+        id,
+        summary,
+        ts,
+      };
+    },
+  },
+  async run() {
+    const fromDate = this.db.get("fromDate");
+    const toDate = this._getCurrentEpoch();
+    const filter = '!SWKA(ozr4ec2cHE9JK'; // See https://api.stackexchange.com/docs/filters
```

Comment:
Interesting, so you created this filter and are just referencing its string representation here?

Reply:
Yes and no :slightly_smiling_face: 
StackExchange offers this "particular" API to create/predefine response filters so that they can be reused. They are also public, so we can reuse filters created by others. In my case, I used their UI to set up a particular filter, and the UI went ahead and handled the creation (or retrieval, probably) of a filter with the specified characteristics:

![stackexchange_filter](https://user-images.githubusercontent.com/5479513/97464883-b4434780-18fe-11eb-95a6-89f0b2de25f4.gif)